xml/libvirt_configuration_gui.xml

```diff
@@ -282,6 +282,16 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+            <term>Enable launch security</term>
+            <listitem>
+              <para>
+                If the &vmhost; supports AMD-SEV technology, activating this
+                option enables secured guest with memory encryption. This option
```

Comment:
It's difficult to parse this sentence for me :-) 
Does it mean that "activating the option enables secure guest access and/with memory encryption"? 
Or "activating the option provides memory encryption to the secure guest"?

Reply:
In fact, this is more complex. This checkbox option is ~10% of what needs to be done to support memory encryption. this Memory encryption in virt-manager is useless as it doesn't do all the requested actions to be able to support it. Probably we should point  to the doc related to this tech: https://documentation.suse.com/sles/15-SP4/single-html/SLES-amd-sev/#article-amd-sev

Reply:
done